Misc/NEWS.d/next/Library/2017-10-03-15-41-08.bpo-31681.sOJMKV.rst

```diff
@@ -0,0 +1 @@
+pkgutil.get_data() does not leak open files
```

Comment:
Not sure if a news entry is useful.

Reply:
It is, we want to have NEWS entries for virtually all commits we make.  Whether those entries will be highlighted in the 'What's new' document is a separate question.